src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/.template.config/template.json

```diff
@@ -21,7 +21,6 @@
   "sourceName": "Company.WebApplication1",
   "preferNameDirectory": true,
   "guids": [
-    "09732173-2cef-46b7-83db-1334bcb079d3", // Tenant ID
```

Comment:
Why is this being removed? The three templates here doesn't seem to be relevant to this change. Is this intentional?

Reply:
Yes. These weren't being used before. It's best to clean these up. Otherwise, you end up with issues where the GUID doesn't get replaced as is the case with the `RazorPagesWeb` template which I also fixed in this PR.

Before this change, `53bc9b9d-9d6a-45d4-8429-2a2761773502` would show up in the templated project output instead of replaced with a random GUID like it should be. You can see evidence of that just by looking at https://grep.app/search?q=53bc9b9d-9d6a-45d4-8429-2a2761773502.

We could try to split this apart and ask for tactics approval for that separately, but I think it's better to save everyone's time and just fix it.

Reply:
Good catch! Thank you!